examples/build.rs

```diff
@@ -240,6 +255,33 @@ fn main() -> anyhow::Result<()> {
     Ok(())
 }
 
+fn generate_dtls_certificate() -> anyhow::Result<(Vec<u8>, Vec<u8>, String)> {
+    let mut param: CertificateParams = CertificateParams::new(vec!["esp32".to_string()]);
+    param.not_before = date_time_ymd(2021, 5, 19);
```

Comment:
Yes we could I didn't see an example where we load a certificate that we use for DTLS. It would still have be done during building until we add support to download certs during runtime